src/coreclr/jit/instrsxarch.h

```diff
@@ -729,14 +729,11 @@ INST1(setge,            "setge",            IUM_WR, 0x0F009D,
 INST1(setle,            "setle",            IUM_WR, 0x0F009E,                                                            Reads_OF       | Reads_SF      | Reads_ZF  )
 INST1(setg,             "setg",             IUM_WR, 0x0F009F,                                                            Reads_OF       | Reads_SF      | Reads_ZF  )
 
-#ifdef TARGET_AMD64
-// A jump with rex prefix. This is used for register indirect
-// tail calls.
-INST1(rex_jmp,          "rex.jmp",          IUM_RD, 0x0020FE,                                                            INS_FLAGS_None)
-#endif
-
-INST1(i_jmp,            "jmp",              IUM_RD, 0x0020FE,                                                            INS_FLAGS_None )
```

Comment:
I noticed that we have the wrong opcode for this and compensate for it in the emitter for some reason, so I have fixed this as well.